### DISCUSSION TITLE
maui windows

### DISCUSSION THREAD

**Discussion by tjiakwokyung28** [dotnet/maui#19581] [General]
remove windows feature, maui running on ios and android only like xamarin, because if you development under android, you having chaotic toolbox position in windows, especially every time it updates from Microsoft

**Comment by SmartmanApps** [comment]
"ios and android only like xamarin" - Xamarin has been running on Windows for many years. Almost all my Xamarin apps were developed as Windows-first.

**Comment by drasticactions** [comment]
This isn't really an "issue" (in a bug that can be "fixed") so I'm moving this to a discussion.

**Comment by tjiakwokyung28** [comment]
write this code
![image](https://github.com/dotnet/maui/assets/104885948/8483ed5a-139b-4141-a5fd-441b6d6dae69)
run on android, then run on windows. you must see the amazing result.

Microsoft must be buy uno then merge to maui.

**Reply by SmartmanApps** [reply]
Try it without XAML (and/or shell) - see [Creating MAUI UI's in C#](https://dev.to/smartmanapps/creating-maui-uis-in-c-1adf) (includes multiple comparisons between Windows and Android) - there are certainly some issues on Windows, but Grid and colours aren't any of them at the moment. Are you on .NET8? Quite a few bugs were fixed in that.

**Reply by tjiakwokyung28** [reply]
yes i'am using net 8, in net 6 windows have minimum problem. use c# for xaml design is very dificully and spending time

**Comment by tjiakwokyung28** [comment]
many bugs make amazing result maui on windows
